meta-lmp-base/recipes-connectivity/docker-network-ref/docker-network-ref/docker-network-ref.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+#
+# Run this script as root to prepare docker-network-ref bridge network for docker
+#
+
+if [ -z "`docker network list -q -f name=docker-network-ref`" ]; then
```

Comment:
Should we allow the user to easily customize the network name?

We could set a variable here which can be given via env at the systemd job (with a proper default value).